devtools/gradle/gradle-application-plugin/src/test/java/io/quarkus/gradle/QuarkusPluginTest.java

```diff
@@ -84,7 +95,67 @@ public void shouldReturnMutlipleOutputSourceDirectories() {
 
     }
 
-    private static final List<String> getDependantProvidedTaskName(Task task) {
+    @Test
+    public void shouldNotFailOnProjectDependenciesWithoutMain() throws IOException {
+        var settingFile = testProjectDir.resolve("settings.gradle.kts");
+        var mppProjectDir = testProjectDir.resolve("mpp");
+        var quarkusProjectDir = testProjectDir.resolve("quarkus");
+        var mppBuild = mppProjectDir.resolve("build.gradle.kts");
+        var quarkusBuild = quarkusProjectDir.resolve("build.gradle.kts");
+        Files.createDirectory(mppProjectDir);
+        Files.createDirectory(quarkusProjectDir);
+        Files.writeString(settingFile,
+                "rootProject.name = \"quarkus-mpp-sample\"\n" +
+                        "\n" +
+                        "include(\n" +
+                        "    \"mpp\",\n" +
+                        "    \"quarkus\"\n" +
+                        ")");
+
+        Files.writeString(mppBuild,
+                "buildscript {\n" +
+                        "    repositories {\n" +
+                        "        mavenLocal()\n" +
+                        "        mavenCentral()\n" +
+                        "    }\n" +
+                        "    dependencies {\n" +
+                        "        classpath(\"org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10\")\n" +
```

Comment:
Could we pass `${kotlin.version}` to the test as a system property so that we don't have to change the source code when `kotlin.version` is updated in the project?